commands.yaml

```diff
@@ -1,270 +1,218 @@
+sites folder copy:
+sites folder get:
+sites folder move:
+sites folder remove:
+sites folder rename:
+sites hidedefaultthemes get:
+sites hidedefaultthemes set:
+sites hubsite connect:
+sites hubsite data get:
+sites hubsite disconnect:
+sites hubsite get:
+sites hubsite register:
+sites hubsite rights grant:
+sites hubsite rights revoke:
+sites hubsite set:
+sites hubsite theme sync:
+sites hubsite unregister:
+sites list add:
+sites list contenttype add:
+sites list contenttype get:
+sites list contenttype remove:
+sites list get:
+sites list label get:
+sites list label set:
+sites list remove:
+sites list set:
+sites list sitescript get:
+sites list view field add:
+sites list view field remove:
+sites list view get:
+sites list view remove:
+sites list view set:
+sites list webhook add:
+sites list webhook get:
+sites list webhook remove:
+sites list webhook set:
+sites listitem add:
+sites listitem get:
+sites listitem record declare:
+sites listitem record undeclare:
+sites listitem remove:
+sites listitem set:
+sites navigation node add:
+sites navigation node get:
+sites navigation node remove:
+sites page add:
+sites page clientsidewebpart add:
+sites page column get:
+sites page control get:
+sites page get:
+sites page header set:
+sites page remove:
+sites page section add:
+sites page section get:
+sites page set:
+sites page text add:
+sites propertybag get:
+sites propertybag remove:
+sites propertybag set:
+sites search:
+sites serviceprincipal grant add:
+sites sp grant add:
+sites serviceprincipal grant get:
+sites sp grant get:
+sites serviceprincipal grant revoke:
+sites sp grant revoke:
+sites serviceprincipal permissionrequest approve:
+sites sp permissionrequest approve:
+sites serviceprincipal permissionrequest deny:
+sites sp permissionrequest deny:
+sites serviceprincipal permissionrequest get:
+sites sp permissionrequest get:
+sites serviceprincipal set:
+sites sp set:
+sites site add:
+sites site appcatalog add:
+sites site appcatalog remove:
+sites site classic add:
+sites site classic get:
+sites site classic remove:
+sites site classic set:
+sites site get:
+sites site inplacerecordsmanagement set:
+sites site groupify:
+sites site set:
+sites sitedesign add:
+sites sitedesign apply:
+sites sitedesign get:
+sites sitedesign remove:
+sites sitedesign rights grant:
+sites sitedesign rights get:
+sites sitedesign rights revoke:
+sites sitedesign run get:
+sites sitedesign run status get:
+sites sitedesign set:
+sites sitedesign task get:
+sites sitescript add:
+sites sitescript get:
+sites sitescript remove:
+sites sitescript set:
+sites storageentity get:
+sites storageentity remove:
+sites storageentity set:
+sites tenant appcatalogurl get:
+sites tenant settings get:
+sites tenant settings set:
+sites term add:
+sites term get:
+sites term group add:
+sites term group get:
+sites term set add:
+sites term set get:
+sites theme apply:
+sites theme get:
+sites theme remove:
+sites theme set:
+sites web add:
```

Comment:
@waldekmastykarz , I am not sure what to do with the `web`. Should it be renamed to `subsite` or something?

Reply:
I think for our audience, `web` makes the most sense